samples/test/config.yaml

```diff
@@ -22,39 +22,47 @@
 # The field `path` corresponds to the test's python module path
 # e.g. if folder path is `samples/test/fail_test.py`, then module path is
 # `samples.test.fail_test`.
-- name: fail
-  path: samples.test.fail_test
-- name: two_step
-  path: samples.test.two_step_test
-- name: after
-  path: samples.test.after_test
 - name: condition
   path: samples.core.condition.condition_test
 - name: nested_condition
   path: samples.core.condition.nested_condition_test
+- name: exit_handler
+  path: samples.core.exit_handler.exit_handler_test
+- name: output_a_directory
+  path: samples.core.output_a_directory.output_a_directory_test
 - name: loop_output
   path: samples.core.loop_output.loop_output_test
 - name: loop_static
   path: samples.core.loop_static.loop_static_test
 - name: loop_parameter
   path: samples.core.loop_parameter.loop_parameter_test
-- name: parameter_with_format
-  path: samples.test.parameter_with_format_test
+- name: loop_parallelism
```

Comment:
D you think we can include the list of supported modes here as well? This way there is no need to have separate test py files alongside the tests.

In the future we could also support pipeline argument sets here.